src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -191,10 +192,24 @@ export class WalletOverviewPage {
    * Returns a promise when initialization is done
    */
   initialize(): Promise<void> {
-    this.getAndSubscribeToTranslations();
     return Promise.all([
       this.getAllWallets(), 
-      this.getAvailableExchanges()]).then<void>();
+      this.getAvailableExchanges()]).then(data => { this.initialized = true; }).catch(data => {
```

Comment:
Why use the catch method? Why not simply pass two function to the then method?

Usually when working with promises I use this form:
```
.then(
    () => { /* SUCCESS! */ },
    (error) => { */ Oh no! Something went wrong! */ }
);
```

The form you use is usually used when the success handler of the promise could also throw an error. In such a case the error will be handled by the final catch. Looking at this code this does not appear to be the case here.

Effectively the catch is the same as doing:
```
.then(undefined, () => { */ Oh no! Something went wrong! /* }
```